script.js

```diff
@@ -0,0 +1,91 @@
+const hamburguer = document.querySelector('.hamburguer');
+const navMenu = document.querySelector('.nav-menu');
+const navLogo = document.querySelector('.nav-logo');
+
+hamburguer.addEventListener('click', () => {
+  hamburguer.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  navLogo.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburguer.classList.remove('active');
+  navMenu.classList.remove('active');
+  navLogo.classList.toggle('active');
+}));
+
+const artistsInfo = [
+  {
+    img: 'images/singuer1.png',
+    name: 'David Guetta',
+    title: 'Techno Artist',
+    info: 'One of the top techno DJ',
+  },
+
+  {
+    img: 'images/singuer2.png',
+    name: 'Sia',
+    title: 'Pop Artist',
+    info: 'One of the top pop singuers',
+  },
+
+  {
+    img: 'images/singuer3.png',
+    name: 'Avenged Sevenfold',
+    title: 'Rock Artist',
+    info: 'One of the top rock bands',
+  },
+
+  {
+    img: 'images/singuer4.png',
+    name: 'Bruno Mars',
+    title: 'Pop Artist',
+    info: 'One of the top pop singuers',
+  },
+
+  {
+    img: 'images/singuer5.png',
+    name: 'Selen Gomez',
+    title: 'Pop Artist',
+    info: 'One of the top pop singuers',
+  },
+
+  {
+    img: 'images/singuer6.png',
+    name: 'Maroon 5',
+    title: 'Pop Artist',
+    info: 'One of the top pop bands',
+  },
+];
```

Comment:
- [ ] Featured speakers section spacing, borders, shadows, and paddings than the original design
![speaker](https://user-images.githubusercontent.com/44745855/176968385-74fde9c7-48a6-45a7-8267-0e77387f94b4.png)